src/common/dsp/effect/CombulatorEffect.cpp

```diff
@@ -0,0 +1,320 @@
+/*
+** Surge Synthesizer is Free and Open Source Software
+**
+** Surge is made available under the Gnu General Public License, v3.0
+** https://www.gnu.org/licenses/gpl-3.0.en.html
+**
+** Copyright 2004-2020 by various individuals as described by the Git transaction log
```

Comment:
No need to redo the PR. But when I touch a file I’ve been turning 20s into 21. Only need to do it when we touch a file of course but just FYI!